awx/ui_next/src/screens/User/User.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { createMemoryHistory } from 'history';
+import { UsersAPI } from '@api';
+import { mountWithContexts, waitForElement } from '@testUtils/enzymeHelpers';
+import mockDetails from './data.user.json';
+import User from './User';
+
+jest.mock('@api');
+
+const mockMe = {
+  is_super_user: true,
+  is_system_auditor: false,
+};
+
+async function getUsers() {
+  return {
+    count: 1,
+    next: null,
+    previous: null,
+    data: {
+      results: [mockDetails],
+    },
+  };
+}
+
+describe.only('<User />', () => {
+  test('initially renders succesfully', () => {
+    UsersAPI.readDetail.mockResolvedValue({ data: mockDetails });
+    UsersAPI.read.mockImplementation(getUsers);
+    mountWithContexts(<User setBreadcrumb={() => {}} me={mockMe} />);
+  });
+
+  test('notifications tab shown for admins', async done => {
+    UsersAPI.readDetail.mockResolvedValue({ data: mockDetails });
+    UsersAPI.read.mockImplementation(getUsers);
+
+    const wrapper = mountWithContexts(
+      <User setBreadcrumb={() => {}} me={mockMe} />
+    );
+    await waitForElement(wrapper, '.pf-c-tabs__item', el => el.length === 5);
+    done();
+  });
+
+  test('should show content error when user attempts to navigate to erroneous route', async done => {
```

Comment:
We can remove the `done` argument and `done()` callback in all of our tests since we're using `async / await`